manpages/fatlabel.8.in

```diff
@@ -94,7 +94,7 @@ between DOS codepage and Language Country/Region:
 .TS
 tab(:);
 c lx.
-\fBCodepage\fR:\fBLanguage Country/Region\fR
+\fBCodepage\fP:\fBLanguage Country/Region\fP
```

Comment:
Note that inside tbl(7) code, avoiding macros is good for portability, so keep the font escapes here.